recipes/lyner/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "lyner" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d15c2ca20facd3d09ce2f9611893c48507849d047e7ff524a7997a197593c167
+
+channels:
+    - conda-forge
+    - bioconda
+#    - plotly
+    - defaults
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - lyner= lyner.main:main
+  script: "{{ PYTHON }} -m pip install 'click_aliases' 'Cluster_Ensembles>=1.16' && {{ PYTHON }} -m pip install . -vv"
```

Comment:
You'll need to add click_aliases and cluster_ensembles to conda-forge first.

Reply:
I just submitted a pull request for click_aliases to conda-forge (and got rid of the cluster_ensembles dependency, since that isn't as easy to package and not strictly needed).
(I accidentally updated this PR already, even though the conda-forge package isn't ready yet, sorry.)

Reply:
No worries, we can just rerun the CI once it's there :)

Reply:
I took the liberty to rerun the tests: click-aliases installs fine. Currently it is missing the LICENSE file, which does not seem to be packaged in the [PyPI](https://pypi.org/project/lyner/#files) package. Adding it to the `data_files` in `setup.py` [should work](https://stackoverflow.com/a/9977956/2862719).